packages/venia-ui/lib/queries/userSubscription.graphql

```diff
@@ -0,0 +1,9 @@
+# expects bearer header to be set via context to return data
+mutation userSubscription($is_subscribed: Boolean!) {
+    updateCustomer(input: { is_subscribed: $is_subscribed }) {
```

Comment:
Naming a little too general, but we try be as close to the constant name defined, which should work here.

```suggestion
    SetNewsletterSubscription(input: { is_subscribed: $isSubscribed }) {
```

Reply:
Magento 2 graphql have not defined `SetNewsletterSubscription` mutation so I have to use `updateCustomer`